src/webassembly.md

```diff
@@ -0,0 +1,11 @@
+# WebAssembly basics
+
+WebAssembly (WASM) is a binary instruction format designed to bring more languages to web browsers (historically, Javascript was the only available choice.) WASM is designed as a portable target for compilation of high-level languages like Rust, enabling it to be used in web applications. WASM has also grown beyond its scope because it can be run in a number of use cases beyond the web, including [nodejs](https://nodejs.dev/en/learn/nodejs-with-webassembly/) or the [Ethereum WebAsssembly Runtime](https://ewasm.readthedocs.io/en/mkdocs/README/)
```

Comment:
Please double-check how the pages look with the aspect-ratio helper. Enable it in [`book.toml`](https://github.com/google/comprehensive-rust/blob/main/book.toml#L22-L27). I fear that the pages contain too much text which results in scrolling.

Reply:
I will have a pass over the whole PR next week and find places where the aspect-ratio is not respected as well as review the new files

Reply:
Great! It'a awesome that you're teaching the regular class this week — I hope this shows you first hand how little space you have on the screen when presenting (remember to ask someone in the back if they can read it) :smile: